packages/catalog-search/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@edx/frontend-enterprise-catalog-search",
-  "version": "10.2.0",
+  "version": "11.0.0",
```

Comment:
Do we actually need to update the version manually or Lerna will handle this?

Reply:
Done